services/surveyService.ts

```
@@ -0,0 +1,356 @@
+              case 'emoji':
+                const emojiOption = question.options?.find(opt => opt.value === answer.answer);
+                answerText = emojiOption ? `${emojiOption.emoji} ${emojiOption.label}` : answer.answer;
+                break;
+              case 'slider':
+                answerText = question.labelFormat ? question.labelFormat(answer.answer) : `${answer.answer}점`;
+                break;
+              case 'yesno':
+                const yesNoOption = question.options?.find(opt => opt.value === answer.answer);
+                answerText = yesNoOption ? `${yesNoOption.emoji} ${yesNoOption.label}` : answer.answer;
+                break;
+              case 'image':
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)